Deliverance/dataobjects/DeliveranceNewsletter.php

```diff
@@ -57,7 +58,7 @@ class DeliveranceNewsletter extends SwatDBDataObject
 	/**
 	 * @var string
 	 */
-	public $template;
+	public $custom_template;
```

Comment:
Existing code use this old `template` field to build an email with a template using that string as folder name from the newsletter directory of the site, and is how all the different designs and visual headers work up to now.

I'm keeping it because I want to continue that behaviour for one-off emails. It's intentionally only editable by direct db access as it's only for us to use when building new newsletters for them, and will become unused over time.